src/components/package-show/package-show.js

```diff
@@ -17,11 +17,11 @@ export default function PackageShow({
   vendorPackage,
   packageTitles,
   toggleRequest,
-  toggleSelected,
-  queryString
+  toggleSelected
 }, { intl, router }) {
   let packageRecord = vendorPackage.content;
-  let historyState = router.history.location.state;
+  const historyState = router.history.location.state;
+  const queryString = router.history.location.search;
```

Comment:
This won’t work when we get into paginating related resources because `location.search` is the entire query param string from the url. 

We’d have to parse it with `query-string` to get the actual `?search` param. 

Since the search route is always rendered, and always parses the params anyway, maybe we could alter the router context to include the parsed params. That would also be helpful for pagination later on.

Reply:
Is the router context different to `router.route.location.search`?

Reply:
Exactly. We basically want to know if we're in a 3-pane layout or not. And that is currently determined by the presence of `searchType` in the search route. Other parts of the query string may eventually relate to paginating packages within the vendor detail pane for example, regardless of whether or not we're in a 3-pane layout.

So if we're on a single-pane detail view on page 2 of the related resources, the `location.search` will still be `?page=2` or what have you, but we still want to show the back button.